react-ai-chatbot/src/components/SettingsModal/SettingsModal.jsx

```
@@ -1,36 +1,40 @@
+                    <label className={styles.CheckboxLabel}>
+                        <input
+                            type="checkbox"
+                            checked={useWebSearch}
+                            onChange={(e) => setUseWebSearch(e.target.checked)}
+                            className={styles.Checkbox}
+                        />
+                        Use web search tool
+                    </label>
+                </div>
+                <div className={styles.Footer}>
+                    <button className={styles.CloseButton} onClick={onClose}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)